### DISCUSSION TITLE
Problems combining wildcards

### DISCUSSION THREAD

**Discussion by temporaryspastic** [adieyal/sd-dynamic-prompts#555] [Q&A]
I have been having difficulty combining any wildcards with other wildcards or any other prompts.  

For example:
When I am using the following prompt:
" __ __bird__ __ ",  I would get results of different birds that are in the respective wildcard text file, ie. "wood duck" as per the image description.
![bird test](https://github.com/adieyal/sd-dynamic-prompts/assets/139179959/5781a9c3-6a8f-410f-95a4-c59d7b99815b)

However, when I add any other text to the prompt other than the solo wildcard, it no longer accesses the text file and it ignores the wildcard parsing cue altogether.  For example inputting "red __bird__" would just generate "red bird".  Or inputting "__ __color__ __ __ __bird__ __" would generate "color bird".  "__ ____color__ __ __ __bird__ __ in the style of __ __artist__ __" would yield "color bird in the style of artist".
![bird test 2](https://github.com/adieyal/sd-dynamic-prompts/assets/139179959/07976bab-e279-45be-9dee-c264ce5eb53f)

![bird test 3](https://github.com/adieyal/sd-dynamic-prompts/assets/139179959/32a1c342-866e-4196-9c4b-28b2db1aa3bc)

I'm not sure if it's a setting that I need to adjust or what, but any help would be much appreciated.  Thanks!

**Comment by akx** [comment]
In your screenshots, you have "Dynamic Prompts enabled" turned off, so Dynamic Prompts is, well, not doing anything. Some other extension you may have installed may be doing its thing.

**Reply by temporaryspastic** [reply]
Hi.  Thanks for the response. I should've mentioned that I've tried it with Dynamic Prompts enabled and disabled but still with the same results.  That's a good thought about maybe disabling some other extensions that might be interfering. I'll try it in the morning!

**Reply by temporaryspastic** [reply]
Unfortunately it seems like switching off every other extension but the wildcards one doesn't seem to help =/
